### PR TITLE
Asynchronously load Persona to avoid blocking JS execution

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

I had to switch to Persona API, because django_browserid API was hit by popup blocker (and I didn't investigate further).

### CODE REVIEW

File: pontoon/base/static/js/main.js [hunk 1]
```diff
@@ -435,50 +435,61 @@ $(function() {
   });
 
   // Profile menu
-  // Register handlers to prep django-browserid before binding menu.
-  django_browserid.registerWatchHandlers(function() {
```
Comment (jotes) [inline pontoon/base/static/js/main.js]: I guess we could remove browserid/api.js from the settings/base.py

File: pontoon/base/static/js/main.js [hunk 2]
```diff
@@ -435,50 +435,61 @@ $(function() {
...
+            url: info.loginUrl,
+            type: 'POST',
+            data: {
+              csrfmiddlewaretoken: csrfToken,
+              assertion: verifyResult
+            },
+            success: function(data) {
+              window.location.reload();
+            },
+            error: function(data) {
```
Comment (jotes) [inline pontoon/base/static/js/main.js]: I think we could report any errors that are happening here. Look at: https://raygun.io/raygun-providers/javascript

File: pontoon/base/static/js/main.js [hunk 3]
```diff
@@ -435,50 +435,61 @@ $(function() {
...
-    });
-
-    $('ul.links > li#sign-in').click(function (e) {
-      e.preventDefault();
-      signIn();
-    });
+    try {
+      navigator.id.request(info.requestArgs);
+    }
+    catch(ex) { }
```
Comment (jotes) [inline pontoon/base/static/js/main.js]: The same here. Also, i think it's a case of Diaper pattern -> c2.com/cgi/wiki?PokemonExceptionHandling .

**Comment by jotes** [comment]
r+ :) This works perfectly in Firefox and Chrome.

File: pontoon/base/static/js/main.js (hunk 1 above)
Comment (mathjazz) [inline pontoon/base/static/js/main.js]: Good idea!

**Pull request merged**